src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -272,6 +265,37 @@ void getMultipleHearingLocations_shouldReturnCorrespondingMultipleEpimsIdForVenu
 
     }
 
+    @DisplayName("Regional hearing location Test")
```

Comment:
Can you describe the test, what method is this unit test for? What are the inputs that cause what outcome?